### PR TITLE
Magic Mathematician Setup

### PR DESCRIPTION
**Pull request by roodjinocherilus**
In this Pull Request I have set up a React Project using "create-react-app"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,184 @@
...
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Movies] <a name="about-project"></a>
+
+**Setting Up A React App**
+# 📖 [project presentation](https://youtu.be/lAMemCTVB4A)
```
Comment (1GORDON) [inline README.md:38]: Great job @roodjinocherilus 🤝

- [ ] [Optional] Kindly remove this project presentation video link from your readme file as it is not relevant to this project.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @roodjinocherilus ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — COMMENTED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @roodjinocherilus,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟

wiserhood-cheers

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:

## [Optional] suggestions:
- [ ] One minor suggestion for improvement is to choose a more descriptive branch name that reflects the action or feature you are working on. For example, instead of setup, you could use created-setup or added-setup. This will help you and other developers to keep track of the changes and the purpose of each branch. You can find more information about branch naming conventions here: https://docs.github.com/en/get-started/using-git/managing-branches
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @topeogunleye in your question so I can receive the notification

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — COMMENTED** [review]
# Hi @roodjinocherilus, 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that is already merged. Kindly submit an open PR so it can be reviewed accordingly 😉 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Pull request merged**